modules/map/hooks/useScreenCenter.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react'
+import { useWindowDimensions } from 'react-native'
+import { useSafeAreaFrame } from 'react-native-safe-area-context'
+
+interface Props {
+  safeArea?: boolean
+  scale?: boolean
+}
+
+export const useScreenCenter = (props?: Props) => {
```

Comment:
Name `props` should be used only with components.

Either use two separate arguments  `useScreenCenter(safeArea?: boolean, scale?: boolean)`
or it is also okay to have an "opstions" object as argument, but name it differently.